### PR TITLE
Enable splitting archive on SeriesInstanceUID [GEAR-202]

### PR DESCRIPTION
**Pull request by NPann**
* Expose configuration option to split archive based on SeriesInstanceUID (and make it default)
* Perform rounding on ImagePatientOrientation before splitting (fix erroneous split when ImagePatientOrientation are alike)
* Update filename renaming based on a sanitized seriesdescription and seriesnumber

### CODE REVIEW

**Comment by NPann** [comment]
Version up to 2.4.0 only. Let me know if that needs a major bump.

File: utils/dicom/dicom_archive.py [hunk 1]
```diff
@@ -220,6 +239,12 @@ def contains_embedded_localizer(self):
         iop_value_list = self.dicom_tag_value_list('ImageOrientationPatient')
         # Convert to list of tuples so it's hashable for set
         iop_tuple_list = make_list_items_hashable(iop_value_list)
+
+        # apply some rounding
```
Comment (lmperry) [inline utils/dicom/dicom_archive.py]: A bit more explanation here would be nice. I know what's going on as we've talked about it, but in a month or two....

File: run.py [hunk 2]
```diff
@@ -760,6 +760,24 @@ def split_embedded_localizer(dcm_archive_path, output_dir, force=False):
             os.sys.exit(0)
 
 
+def split_seriesinstanceUID(dcm_archive_path, output_dir, force=False):
+    with dicom_archive.make_temp_directory() as tmp_dir:
+        dcm_archive_obj = dicom_archive.DicomArchive(dcm_archive_path, tmp_dir, dataset_list=True, force=force)
+        if dcm_archive_obj.contains_different_seriesinstanceUID():
+            log.info('Splitting embedded Series...')
```
[the remark this answers is not in the document]
Comment (NPann) [reply run.py]: Addressed in b7d4ba3.

File: manifest.json [hunk 3]
```diff
@@ -43,7 +48,7 @@
     "force_dicom_read": {
       "description": "Force pydicom to read the input file. This option allows files that do not adhere to the DICOM standard to be read and parsed. (Default=False)",
       "type": "boolean",
-      "default": false
+      "default": true
```
[the remark this answers is not in the document]
Comment (NPann) [reply manifest.json]: Addressed in b7d4ba3.

**Pull request merged**